packages/caml-mode/caml-mode.4.02/opam

```diff
@@ -20,6 +20,11 @@ remove: [
   [make "uninstall-ocamltags" "BINDIR=%{bin}%"]
 ]
 depends: []
+depexts: [
+  [["debian"] ["emacs"]]
+  [["ubuntu"] ["emacs"]]
+  [["macosx"] ["emacs"]]
+]
```

Comment:
thanks! If you could add `[["centos"]["emacs-nox"]]` then I think we have the majority covered.  You can get rid of the `macosx` line as its part of the base distribution there.